<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rna_app</class>
 <widget class="QWidget" name="rna_app">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RNA-Seq</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="data_tab">
      <attribute name="title">
       <string>Data</string>
      </attribute>
      <widget class="QFrame" name="data_frame">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>272</width>
         <height>244</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QLabel" name="data_frame_lbl">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>231</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Load counts file</string>
        </property>
       </widget>
       <widget class="QPushButton" name="load_btn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Data</string>
        </property>
       </widget>
       <widget class="QLabel" name="selected_lbl">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>231</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Selected file:</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="metadata_tab">
      <attribute name="title">
       <string>Metadata</string>
      </attribute>
      <widget class="QFrame" name="metadata_frame">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>571</width>
         <height>261</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QLabel" name="metadata_lbl">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>231</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Metadata</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="metadata_tbl">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>541</width>
          <height>151</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Sample</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Condition</string>
         </property>
        </column>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>210</y>
          <width>301</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="load_metada_btn">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="save_metadata_btn">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="de_tab">
      <attribute name="title">
       <string>DE</string>
      </attribute>
      <widget class="QFrame" name="de_frame">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>10</y>
         <width>272</width>
         <height>244</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QLabel" name="de_frame_lbl">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>231</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Differential Expression</string>
        </property>
       </widget>
       <widget class="QPushButton" name="de_btn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Run DE</string>
        </property>
       </widget>
       <widget class="QLabel" name="de_status_llbl">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>231</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="batch_frame">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>272</width>
         <height>244</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QLabel" name="batch_frame_lbl">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>231</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Batch</string>
        </property>
       </widget>
       <widget class="QComboBox" name="condition1_combo">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>221</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="condition1_lbl">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Condition 1:</string>
        </property>
       </widget>
       <widget class="QLabel" name="condition2_lbl">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Condition 2:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="condition2_combo">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>221</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="batch_err_lbl">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>221</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
